.github/workflows/TestEnvDockerfile

```diff
@@ -1,5 +1,5 @@
-ARG BASE_BASE_IMAGE=bash:5.2
-FROM $BASE_BASE_IMAGE
+ARG BASE_BASH_IMAGE=bash:5.2
+FROM $BASE_BASH_IMAGE
```

Comment:
It seems that there was a typo. Test for each Bash version was running **5.2**.

Reply:
Oh, The fix of this should be merged as possible as fast, shouldn't?

Reply:
#704 created